<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropMatrixMatcher</class>
 <widget class="QMainWindow" name="DropMatrixMatcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>702</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>762</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1250</width>
     <height>650</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>341</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="loadImage">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LoadSelectedPattern">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Select Drop Matrix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadPattern">
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Load extern Drop Matrix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="patternLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>300</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>Pattern Label</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="brightness">
           <property name="text">
            <string>brightness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="brightnessSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="contrast">
           <property name="text">
            <string>contrast</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="contrastSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <widget class="QPushButton" name="compareButton">
         <property name="text">
          <string>detect match</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="sensitivityValueText"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QComboBox" name="comboBox">
         <property name="currentText">
          <string>Method 1</string>
         </property>
         <item>
          <property name="text">
           <string>Method 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Method 1 - normed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Method 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Method 2 - normed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Method 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Method 3 - normed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Test Method A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Test Method B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Test Method B - normed</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>sensitivity</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDial" name="dial">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>load to DB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>400</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="ProLabel" name="imageLabel_1">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>0</y>
           <width>871</width>
           <height>541</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>500</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Image Label</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <widget class="ProLabel" name="imageLabel_2">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>0</y>
           <width>871</width>
           <height>500</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>500</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Image Label</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="greyView">
           <property name="text">
            <string>grey view</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioC">
           <property name="text">
            <string>cyan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioM">
           <property name="text">
            <string>magenta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioY">
           <property name="text">
            <string>yellow</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filterButton">
           <property name="text">
            <string>filter</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="brightness_2">
           <property name="text">
            <string>brightness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="brightnessSlider_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="contrast_2">
           <property name="text">
            <string>contrast</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="contrastSlider_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProLabel</class>
   <extends>QLabel</extends>
   <header>prolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
